Client.UnitTests/CancelProcessInstanceTest.cs

```diff
@@ -18,6 +19,7 @@ public async Task ShouldSendRequestAsExpected()
             {
                 ProcessInstanceKey = 12113
             };
+            TestService.Reset();
```

Comment:
:question: For what these resets have been added?

Reply:
I have changed the logic of these tests and share the same server across all. The execution is now way more faster. But I need to reset its state, otherwise the expectation is polluted by previous methods.